### PR TITLE
Add workflow to bump versions

### PR DESCRIPTION
**Pull request by jtpio**
As discussed with @wolfv it would be great to have some automation on this repo to bump the versions of the packes.

This PR adds the workflow and script from https://github.com/mamba-org/boa-forge.

### CODE REVIEW

**Comment by martinRenou** [comment]
Amazing! Thanks for opening this. 

Would be great to have an auto-merge bot as well at some point for those automatic PRs, if the CI is green

**Comment by jtpio** [comment]
> Would be great to have an auto-merge bot as well at some point for those automatic PRs, if the CI is green

Sounds good. I think this could be a nice follow-up step.

File: determine_versions.py [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+                raw_yaml["source"][0]["sha256"] = sha256_hash_for_version
+            else:
+                raw_yaml["source"]["sha256"] = sha256_hash_for_version
+        raw_yaml["build"]["number"] = 0
+    return raw_yaml
+
+if __name__ == "__main__":
+    import glob
+    loader = get_yaml_loader(typ="rt")
+    recipes = glob.glob("recipes_emscripten/**/recipe.yaml")
```
Comment (jtpio) [inline determine_versions.py:642]: Adapted the script from https://github.com/mamba-org/boa-forge/blob/d000999ea95a6b95d5f497e00479749f7a1b24c7/determine_versions.py#L642 to search in the `recipes_emscripten` folder.

File: determine_versions.py [hunk 2]
```diff
@@ -0,0 +1,648 @@
+# License: BSD3
```
Comment (martinRenou) [inline determine_versions.py:1]: Could `bump_recipies_versions.py` be a better name for this file? As it really update the recipe.yaml files?

Also, I've seen this yesterday https://twitter.com/condaforge/status/1647933355877445633?s=20, is there a way to do something similar here?

Comment (jtpio) [reply determine_versions.py:1]: > Could `bump_recipies_versions.py` be a better name for this file? As it really update the recipe.yaml files?

Sure :+1: 

> Also, I've seen this yesterday https://twitter.com/condaforge/status/1647933355877445633?s=20, is there a way to do something similar here?

That would be nice. Although likely to work on conda feedstock only? Not sure which tool allows for doing something like this but if we find it we could indeed use it in this repo (maybe @wolfv knows).

**Review by martinRenou — APPROVED** [review]
Thanks! Let's try it

**Pull request merged**